src/UI/Implementation/Crawler/Entry/ComponentEntry.php

```diff
@@ -451,13 +451,25 @@ public function getExamplesPath(){
 		if(!$this->examples_path){
 			$path_componants = str_replace("Component","examples",$this->getPath())
 					."/".str_replace(" ", "", $this->getTitle());
-			$path_array = array_unique(explode ("/",$path_componants));
+			$path_array = self::array_iunique(explode ("/",$path_componants));
```

Comment:
Oh sorry...this was a dirty fix because the examples for the roundtrip modals didn't show up. The reason was that I used `RoundTrip` as interface/class names, and the example logic assumed `Roundtrip`.